module/VuFind/src/VuFind/Cover/Router.php

```diff
@@ -125,23 +125,20 @@ public function getMetadata(
             return false;
         }
 
+        if (!is_array($thumb)) {
```

Comment:
And if we're adding comments, maybe it's worth providing some context for this check as well:

```suggestion
        // If $thumb is not an array, it is a full URL to a thumbnail image.
        if (!is_array($thumb)) {
```